src/verilog/expr2verilog.cpp

```diff
@@ -863,6 +938,26 @@ std::string expr2verilogt::convert_next_symbol(
 
 /*******************************************************************\
 
+Function: expr2verilogt::convert_hierarchical_identifier
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+std::string expr2verilogt::convert_hierarchical_identifier(
+  const hierarchical_identifier_exprt &src,
+  unsigned &precedence)
+{
+  precedence = 22;
```

Comment:
Could you please add a comment for this magic number?

Reply:
#591